packages/express-backend/src/server.js

```diff
@@ -120,6 +120,22 @@ app.get('/users/:userId/items', async (req, res) => {
   res.status(statusCode).send(items || [])
 })
 
+app.get('/users/:userId/items-bought', async (req, res) => {
+  const userId = req.params.userId
+  const items = await Database.findBoughtItemsByUserId(userId)
+  const statusCode = items === undefined ? 500 : 200
+
+  res.status(statusCode).send(items || [])
+})
+
+app.post('/users/:userId/items-bought', async (req, res) => {
```

Comment:
Since an item will be officially "sold" by the person who posted it, maybe this endpoint should be something like PUT `/users/:userId/items/:itemId` where the receiving user's id is in the request body?

This is mainly a concern because when writing authorization on this endpoint as it is now, the authorization function would have to call the backend to get the item of interest and then check if the `seller_id` on that item matches the caller's authorization token.